pkg/cmd/pipeline/describe_test.go

```diff
@@ -91,11 +91,11 @@ func TestPipelinesDescribe_with_run(t *testing.T) {
 	}
 
 	expected := []string{
-		"Name: pipeline",
+		"Name:   pipeline",
 		"\nResources",
-		"No resources found\n",
+		"No resources\n",
 		"Tasks",
-		"No tasks found\n",
+		"No tasks\n",
```

Comment:
The same with [the comment](https://github.com/tektoncd/cli/pull/131/files#r298817613). Since `describe pipelinerun` says `No resources`, I remove `found` to print the same message.

https://github.com/tektoncd/cli/blob/master/pkg/cmd/pipelinerun/describe.go#L52